src/geode/model/helpers/component_mesh_vertices.cpp

```diff
@@ -49,9 +49,18 @@ namespace
                     OPENGEODE_ASSERT( cmv0.vertex != cmv1.vertex,
                         "[component_mesh_vertex_pairs] Vertices should be "
                         "different" );
-                    result[cmv0.component_id].emplace_back(
-                        std::array< geode::index_t, 2 >{
-                            cmv0.vertex, cmv1.vertex } );
+                    if( result.contains( cmv0.component_id ) )
```

Comment:
Using find and adding directly to the iterator would be faster than contains + at